scrapers/NoelAlejandro.yml

```diff
@@ -0,0 +1,37 @@
+name: NoelAlejandro
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - noelalejandrofilms.com/product
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //h1[@id="title"]
+      Image:
+        selector: //video[@id="product-img"]/@poster
+      Date:
+        selector: //li[contains(.,"Year:")]
+        postProcess:
+          - replace: 
+              - regex: ^\s*Year:\s*
+                with: ''
+          - parseDate: 2006
```

Comment:
This grabs the date published from the embedded JSON
```yaml
      Date:
        selector: //script[@class="yoast-schema-graph"]
        postProcess:
          - replace: 
              - regex: .+datePublished\"\:\"(\d{4}-\d{2}-\d{2}).+
                with: $1
```

Reply:
And so it does.  Not sure why I missed that. Thanks for the assist.